content/posts/networkx/outreachy2023/contribution-phase/index.md

```diff
@@ -0,0 +1,43 @@
+
+2. _Take part in the contribution phase:_ Once your initial application is approved you will be able to see all the projects. Finding the right project for you is important and also very challenging. One can feel tempted to contribute to multiple projects but unless you have a lot of free time I don’t feel like it's the best option. Smaller and more constant contributions are the way to go. The contribution phase is not about introducing huge contributions but rather an opportunity to interact with the community, learn about the project, and gain new skills. Definitely finding the right project for you is key and depends a lot on how much time are you willing to put into it and your current skills.
+
+For more information about Outreachy go to: <https://www.outreachy.org/>
+
+## My experience at NetworkX during the contribution phase
+
+If this is your first time contributing to an open-source project you may feel overwhelmed. Understanding an almost 20-year-old project like NetworkX can feel like it's gonna take forever but don't worry I have some tips that may be handy for you during the contribution phase.
+
+- _Learn about the project:_ Understanding the project is a process that may take some time. Don’t rush it! You don’t need to understand all codebase in a day. The most important things that you need to know only will take you a few hours to go through: Learn about the project mission and values,
+  community rules, and contribution process. In NetworkX all you need to know is here: <https://networkx.org/documentation/stable/developer/index.html>
+
+- _Start contributing right away:_ You don't need to understand every part of the project to make valuable contributions. Start small and use that experience to level up your contributions. At the beginning of the contribution phase, some good first issues are added. Work on them first and then start opening your own issues (Don’t forget to link your PR with the issues so they can be automagically closed). Also, record your contributions on the Outreachy website as you submit them. I recorded all contributions at the end and took me a lot of time.
```

Comment:
Is the contribution list public? It might be interesting to make a link? Can also be a GitHub search

Reply:
There's a list that is public but it's just easier to access the PR using a GitHub search. It'll be a nice add. Thanks!